drivers/vulkan/rendering_device_vulkan.cpp

```diff
@@ -8394,6 +8394,7 @@ void RenderingDeviceVulkan::_free_internal(RID p_id) {
 		b.allocation = index_buffer->allocation;
 		b.buffer = index_buffer->buffer;
 		b.size = index_buffer->size;
+		b.buffer_info = VkDescriptorBufferInfo();
```

Comment:
I can't find an explicit definition for this constructor so not 100% sure what it does, but I think in other places we'd done default initialization with `= {}`.